api/odigos/v1alpha1/instrumentatioconfig_types.go

```diff
@@ -41,16 +42,76 @@ type SdkConfig struct {
 
 	// configurations for the instrumentation libraries the the SDK should use
 	InstrumentationLibraryConfigs []InstrumentationLibraryConfig `json:"instrumentationLibraryConfigs"`
+
+	// HeadSamplingConfig is a set sampling rules.
+	// This config currently only applies to root spans.
+	// In the Future we might add another level of configuration base on the parent span (ParentBased Sampling)
+	HeadSamplingConfig HeadSamplingConfig `json:"headSamplerConfig,omitempty"`
+}
+
+// 'Operand' represents the attributes and values that an operator acts upon in an expression
+type AttributeCondition struct {
+	// attribute key (e.g. "url.path")
+	Key attribute.Key `json:"key"`
+	// currently only string values are supported.
+	Val string `json:"val"`
+	// The operator to use to compare the attribute value.
+	Operator Operator `json:"operator"`
+}
+
+type Operator string
+
+const (
+	Equals    Operator = "equals"
+	NotEquals Operator = "notEquals"
+	Contain   Operator = "contain"
```

Comment:
I wonder, since this operator cannot be implemented with eBPF, perhaps we can omit it now and consider it in the future when we meet a real usecase. then we need to make sure it can only be used with non eBPF SDKs.

Reply:
nit: I think it should be `Contains` (with an s), or drop the s in the equals/notEquals